hack/dev/Dockerfile.okteto

```diff
@@ -156,17 +138,17 @@ RUN cd /tmp && curl -fsSL -o helm.tar.gz "${HELM3_URL}" \
   && ln -s "${KOTS_HELM_BIN_DIR}/helm3" "${KOTS_HELM_BIN_DIR}/helm" \
   && rm -rf helm.tar.gz linux-amd64
 
-# Install s3cmd
-#ENV S3CMD_VERSION=2.1.0
-#ENV S3CMD_URL=https://github.com/s3tools/s3cmd/releases/download/v${S3CMD_VERSION}/s3cmd-${S3CMD_VERSION}.tar.gz
-#RUN cd /tmp && curl -fsSL -o s3cmd.tar.gz "${S3CMD_URL}" \
-#  && curl -fsSL -o s3cmd.tar.gz.asc "${S3CMD_URL}.asc" \
-#  && gpg --keyserver keyserver.ubuntu.com --recv-keys 0x0d37a8f4a5d183d5541d85d9241769189ac3d00b \
-#  && cat s3cmd.tar.gz.asc \
-#  && gpg --verify s3cmd.tar.gz.asc s3cmd.tar.gz \
-#  && tar -xzvf s3cmd.tar.gz \
-#  && mv s3cmd-${S3CMD_VERSION}/s3cmd /usr/local/bin/s3cmd \
-#  && mv s3cmd-${S3CMD_VERSION}/S3 /usr/local/bin/S3 \
-#  && rm -rf s3cmd.tar.gz s3cmd.tar.gz.asc s3cmd-${S3CMD_VERSION}
+RUN go install github.com/go-delve/delve/cmd/dlv@v1.8.0
+
+ENV PROJECTPATH=/go/src/github.com/replicatedhq/kots
+WORKDIR $PROJECTPATH
+
+COPY go.mod go.sum ./
+RUN --mount=target=/go/pkg/mod,type=cache go mod download
```

Comment:
@syntastical we are going back and forth in repos with the target path here. I'd like to figure out the definite right solution here. Let's chat with Okteto team to get their recommendation? 

I've seen some projects moving to this path, and some moving away from it. I think we need to all be on the same page and in agreement about how we want gomod cached.